ipv6-in-vnet/azuredeploy.json

```diff
@@ -0,0 +1,370 @@
+    "imageSku": "2016-Datacenter"
+  },
+  "resources": [
+    {
+      "apiVersion": "[variables('ApiVersion')]",
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "[concat(variables('publicipName'), copyindex())]",
+      "copy": {
+        "name": "publicIpLoop",
+        "count": "[variables('numberOfInstances')]"
+      },
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "publicIPAllocationMethod": "Dynamic"
+      }
+    },
+	{
+      "apiVersion": "[variables('ApiVersion')]",
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "lbpublicip",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "publicIPAllocationMethod": "Dynamic"
+      }
+    },
+	{
+      "apiVersion": "[variables('ApiVersion')]",
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "lbpublicip-v6",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "publicIPAllocationMethod": "Dynamic",
+		"publicIPAddressVersion": "IPv6"
+      }
+    },
+    {
+      "type": "Microsoft.Compute/availabilitySets",
+      "name": "[variables('availabilitySetName')]",
+      "apiVersion": "2016-04-30-preview",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "platformFaultDomainCount": "1",
+        "platformUpdateDomainCount": "1",
+        "managed": "true"
+      }
+    },
+    {
+      "apiVersion": "2015-06-15",
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[variables('newStorageAccountName')]",
+      "location": "eastus2",
+      "properties": {
+        "accountType": "Standard_LRS"
+      }
+    },
+    {
+      "apiVersion": "[variables('ApiVersion')]",
+      "name": "loadBalancer",
+      "type": "Microsoft.Network/loadBalancers",
+      "location": "[resourceGroup().location]",
+	   "dependsOn": [
+        "Microsoft.Network/publicIPAddresses/lbpublicip",
+		"Microsoft.Network/publicIPAddresses/lbpublicip-v6"
+      ],
+      "properties": {
+        "frontendIpConfigurations": [
+          {
+            "name": "LBFE",
+            "properties": {
+              "publicIPAddress": {
+                "id": "[resourceId('Microsoft.Network/publicIPAddresses', 'lbpublicip')]"
+              }
+            }
+          },
+		  {
+            "name": "LBFE-v6",
+            "properties": {
+              "publicIPAddress": {
+                "id": "[resourceId('Microsoft.Network/publicIPAddresses','lbpublicip-v6')]"
+              }
+            }
+          }
+        ],
+        "backendAddressPools": [
+          {
+            "name": "LBBAP"
+          },
+		  {
+            "name": "LBBAP-v6"
+          }
+        ],
+        "loadBalancingRules": [
+          {
+            "properties": {
+              "frontendIPConfiguration": {
+                "id": "[concat(resourceId('Microsoft.Network/loadBalancers', 'loadBalancer'), '/frontendIpConfigurations/LBFE')]"
```

Comment:
remove concat and use resourceId (here and below)

resourceId('Microsoft.Network/loadBalancers/frontendIpConfigurations', 'loadBalancer', 'LBFE')